app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module API
+  module V1
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/3bdd1603c87059b5dda5aabfb14999fb437bbf29/app/controllers/api/v1/users_controller.rb#L4"></span>Has the name ‘v1’</td>
    </tr>
  </tbody>
</table>